components/movementsensor/gpsnmea/serial.go

```diff
@@ -57,8 +57,8 @@ func NewSerialGPSNMEA(ctx context.Context, name resource.Name, conf *Config, log
 	}
 	baudRate := conf.SerialConfig.SerialBaudRate
 	if baudRate == 0 {
-		baudRate = 9600
-		logger.Info("SerialNMEAMovementSensor: serial_baud_rate using default 9600")
+		baudRate = 115200
+		logger.Info("SerialNMEAMovementSensor: serial_baud_rate using default 115200")
```

Comment:
I wonder how the baudRate 9600 worked before. Was it for any other chip? Should we consider checking 9600 just in case?

Reply:
good catch, default baud rate was for this model: [link ](https://www.google.com/search?client=firefox-b-1-d&q=sparkfun+gps#oshopproduct=gid:5670059868232628683,mid:576462723767279377,oid:11136194096792245361,iid:2788386993970223846,rds:UENfNTY3MDA1OTg2ODIzMjYyODY4M3xQUk9EX1BDXzU2NzAwNTk4NjgyMzI2Mjg2ODM%3D,pvt:hg&oshop=apv&pvs=0) the rtk gps is: [link](https://www.sparkfun.com/products/15136)

Reply:
also many gpsnmea models, we won't address this properly until we have a baud rate discover feature for sensors.

Reply:
Changing this to 38400 since that's what the datasheet says the default value is, but just confirmed that 9600 and 115200 also work for me.